src/modules/RA/Viewpoint/views/List.js

```diff
@@ -1,13 +1,80 @@
 import React from 'react';
-import { List } from 'react-admin';
+import {
+  Filter,
+  List,
+  SelectArrayInput,
+  SelectInput,
+  TextInput,
+} from 'react-admin';
 
 import CommonBulkActionButtons from '../../../../components/react-admin/CommonBulkActionButtons';
 import GridList from '../../../../components/react-admin/GridList';
+import { fetchFilterOptions } from '../../ra-modules';
+import { connectAppProvider } from '../../../../components/AppProvider';
+
+
+const ListFilters = props => {
+  const [filters, setFilters] = React.useState([]);
+
+  const setFiltersList = async () => {
+    const filterOptions = await fetchFilterOptions();
+
+    // To generate the select input list to render, first check that the filters in props.properties
+    // are also set in the retrieved filter options from the backend. After that we can generate the
+    // corresponding choices list to each select and generate its properties to be used on render.
```

Comment:
Prefer block comment (`/* */`) for (pseudo)doc.
Line comment (`//`) should be kept for enabled/disabled lines.

```js
    /*
      To generate the select input list to render, first check that the filters in props.properties
      are also set in the retrieved filter options from the backend. After that we can generate the
      corresponding choices list to each select and generate its properties to be used on render. 
    */
```

Reply:
:+1: